Sources/ProjectDescription/Dependencies/SwiftPackageManagerDependencies.swift

```diff
@@ -14,21 +14,29 @@ public struct SwiftPackageManagerDependencies: Codable, Equatable {
     // Additional settings to be added to targets generated from SwiftPackageManager.
     public let targetSettings: [String: SettingsDictionary]
 
+    /// Options for generating automatic schemes and resource assessors for each generated project (from a swift package)
+    public let projectConfigurations: [String: ProjectDescription.Project.ProjectConfiguration]
+
     /// Creates `SwiftPackageManagerDependencies` instance.
     /// - Parameter packages: List of packages that will be installed using Swift Package Manager.
     /// - Parameter productTypes: The custom `Product` types to be used for SPM targets.
     /// - Parameter baseSettings: Additional settings to be added to targets generated from SwiftPackageManager.
     /// - Parameter targetSettings: Additional settings to be added to targets generated from SwiftPackageManager.
+    /// - Parameter projectConfigurations: Options to control automatic schemes and resource accessors generation
+    ///  for Swift Packages i.e ["package_name":  ProjectConfiguration]
```

Comment:
Let's be consistent with the others
```suggestion
Suggested change 
    /// - Parameter projectConfigurations: Custom project configurations to be used for projects generated from SwiftPackageManager.
```